src/app/components/elements/Voting.jsx

```diff
@@ -141,23 +141,23 @@ class Voting extends React.Component {
         };
 
         this.storeSliderWeight = up => () => {
-            const { username } = this.props;
+            const { username, is_comment } = this.props;
             const weight = up
                 ? this.state.sliderWeight.up
                 : this.state.sliderWeight.down;
             localStorage.setItem(
-                'voteWeight' + (up ? '' : 'Down') + '-' + username,
+                'voteWeight' + (up ? '' : 'Down') + (is_comment ? '-comment' : '') + '-' + username,
                 weight
             );
         };
         this.syncSliderWeight = () => {
-            const { username, net_vesting_shares } = this.props;
+            const { username, net_vesting_shares, is_comment } = this.props;
             if (net_vesting_shares > VOTE_WEIGHT_DROPDOWN_THRESHOLD) {
                 const sliderWeightUp = Number(
-                    localStorage.getItem('voteWeight' + '-' + username)
+                    localStorage.getItem('voteWeight' + (is_comment ? '-comment' : '') + '-' + username)
                 );
                 const sliderWeightDown = Number(
-                    localStorage.getItem('voteWeight' + 'Down' + '-' + username)
```

Comment:
We want to keep 'Down' here.

Reply:
`Down` is still there - they just got rid of the unnecessary string concatenation. let's use template literals for these though!